scripts/EN/Cybil

```diff
@@ -1362,37 +1362,37 @@ LoadBackground(0x44); // 0x1b10: 00001069
 HandleInput(); // 0x1b44: 00000010
-ShowText((5164)"¥"You have done well to come here. You know, don't you? That it was my doing that caused... And you were hesitant to come alone, weren't you? But in the end I was able to meet you. It is all thanks to that woman.¥""); // 0x1b48: 0000100c
+ShowText((5164)"¥"You have done well to come here.¥nW2You know, don't you?¥nW2That it was my doing that caused…¥n¥nAnd you were hesitant to come alone, weren't you?¥nW2But in the end I was able to meet you.¥nW2It is all thanks to that woman.¥""); // 0x1b48: 0000100c
 HandleInput(); // 0x1b4e: 00000010
-ShowText((5165)"¥"I don't understand what you are talking about.¥" Cybil's words may or may not have reached her, but that woman began to laugh loudly again. ¥"Hey, tell me. Why are there no people here? Why are all the exits from town blocked?¥""); // 0x1b52: 0000100c
+ShowText((5165)"¥"W3I don't understand what you are talking about.¥"¥nW2Cybil's words may or may not have reached her, W1but that woman began to laugh loudly again.¥n¥nW2¥"Hey, W1tell me.¥nW2Why are there no people here?¥nW2Why are all the exits from town blocked?¥""); // 0x1b52: 0000100c
 HandleInput(); // 0x1b58: 00000010
-ShowText((5166)"¥"I do not know either. But shouldn't you be asking that girl instead?¥" Cybil had a look of ¥"What?¥" on her face. She looked at Cheryl. Cheryl face was tearful, and she shook her head back and forth."); // 0x1b5c: 0000100c
+ShowText((5166)"¥"I do not know either.¥nW2But shouldn't you be asking that girl instead?¥"¥n¥nW2Cybil had a look of ¥"What?¥" W1on her face. She looked at Cheryl.¥nW2Cheryl face was tearful, and she shook her head back and forth."); // 0x1b5c: 0000100c
 HandleInput(); // 0x1b62: 00000010
-ShowText((5167)"Cybil looked back at that woman and shouted at her. ¥"Stop all this nonsense! We have been pursued by monsters and nearly killed. If I was not protecting this child then who knows what would have happened to her by now.¥""); // 0x1b66: 0000100c
+ShowText((5167)"Cybil looked back at that woman and shouted at her.¥nW2¥"Stop all this nonsense!¥nW2We have been pursued by monsters and nearly killed.¥nW1If I was not protecting this child then who knows what would have happened to her by now.¥""); // 0x1b66: 0000100c
 HandleInput(); // 0x1b6c: 00000010
-ShowText((5168)"¥"The path of the Wiseman... the fool guided to this land through astrology. Everything has been preordained. There is nothing that I can do. You are not holding that child's hand. That child is holding yours.¥""); // 0x1b70: 0000100c
+ShowText((5168)"¥"The path of the hermit…W3the fool guided to this land through astrology.¥nW2Everything has been preordained.¥n¥nW2There is nothing that I can do.¥nW2You are not holding that child's hand.¥nW2That child is holding yours.¥""); // 0x1b70: 0000100c
```

Comment:
Changed the wrong wiseman to hermit like it should have always been. (Thanks in part to Persona, didn't even need to look things up.)